Pipfile

```diff
@@ -17,6 +17,10 @@ whitenoise = "*"
 unittest-xml-reporting = "*"
 django = "*"
 django-labs-accounts = "*"
+django-extensions = "*"
+drf-nested-routers = "*"
+"bs4" = "*"
+bleach = "*"
 
 [requires]
-python_version = "3.7"
+python_version = "3"
```

Comment:
Is there any particular reason to switch this? We're running 3.7 in production, so I just wanted to be consistent across development and production environments.

Reply:
Not all development computers have Python 3.7 installed (usually Python 3.5 or Python 3.6). Adding this specific requirement makes Pipenv fail when installing dependencies, even though the app will run perfectly fine with any Python 3.5+. Ideally I would want to limit this a bit further, but this is not currently supported by Pipenv.

The alternatives would be having to deal with managing multiple Python versions on development computers, or changing the version from 3.7 to 3 every time you needed to use Pipenv.

Reply:
Hmm, I see. I want to eventually move us over to docker, potentially even for development so we don't have to manage setting up environments, but for now this makes sense.